cmd/state.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/BOPR/core"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+//  viewState
+func viewState() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "account-info",
+		Short: "returns decoded account info",
```

Comment:
Do we want state here?

```suggestion
		Use:   "state-info",
		Short: "returns decoded state info",
```

Reply:
Sure, fixing